From 406bca78e4979831737bf13c8d990237a82eb5ba Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Mon, 9 Nov 2015 15:30:45 +0100
Subject: [PATCH] uploader: move username and password to the advanced options

Due to possibility configure SSH keys the username and the password became not
so important options therefore were moved from Basic to Advanced options.

Related to rhbz#1289513

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/plugins/report_Uploader.xml.in | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/plugins/report_Uploader.xml.in b/src/plugins/report_Uploader.xml.in
index 8d648eb..db00c23 100644
--- a/src/plugins/report_Uploader.xml.in
+++ b/src/plugins/report_Uploader.xml.in
@@ -19,17 +19,17 @@
             <_note-html>Examples:&#xA;ftp://[user[:pass]@]host/dir/[file.tar.gz]&#xA;scp://[user[:pass]@]host/dir/[file.tar.gz]&#xA;file:///dir/[file.tar.gz]</_note-html>
             <default-value></default-value>
         </option>
-        <option type="text" name="Upload_Username">
-            <_label>User name</_label>
-            <allow-empty>yes</allow-empty>
-            <_description>Use this field if you do not want to have user name in URL</_description>
-        </option>
-        <option type="password" name="Upload_Password">
-            <_label>Password</_label>
-            <allow-empty>yes</allow-empty>
-            <_description>Use this field if you do not want to have password in URL</_description>
-        </option>
         <advanced-options>
+            <option type="text" name="Upload_Username">
+                <_label>User name</_label>
+                <allow-empty>yes</allow-empty>
+                <_description>Use this field if you do not want to have user name in URL</_description>
+            </option>
+            <option type="password" name="Upload_Password">
+                <_label>Password</_label>
+                <allow-empty>yes</allow-empty>
+                <_description>Use this field if you do not want to have password in URL</_description>
+            </option>
             <option type="text" name="http_proxy">
                 <_label>HTTP Proxy</_label>
                 <allow-empty>yes</allow-empty>
-- 
1.8.3.1

